jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -439,80 +444,122 @@ private void checkImageInTargetRegistry() {
                                 results
                                     .builtImagesAndContainerConfigurationPushResults
                                     .get()
-                                    .get(results.builtImages.get().get(0)))
+                                    .values()
+                                    .iterator()
+                                    .next())
                             .get(),
-                        results.builtImages.get().get(0).get())
+                        Verify.verifyNotNull(
+                                results
+                                    .builtImagesAndContainerConfigurationPushResults
```

Comment:
I think we should keep using `builtImages`. I think using `builtImagesAndContainerConfigurationPushResults` to get an `Image` causes confusion and feels like abusing.